readthedocs/api/v3/tests/responses/projects-sync-versions.json

```diff
@@ -0,0 +1 @@
+{"triggered": true}
```

Comment:
I would return 204 with an empty body here to keep consistency with other endpoints.

Reply:
I copied the approach from the build endpoint:

https://github.com/readthedocs/readthedocs.org/blob/e63111ff9415dd135636927782f5bc60c388d85a/readthedocs/api/v3/views.py#L423-L429

Reply:
Huh, interesting. I didn't remember this 👍🏼